### ISSUE TITLE
NUKE + sessionDataBucketAuth == BOOTLOOP

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2264]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.23.7](https://github.com/open-wa/wa-automate-nodejs/compare/4.23.6...4.23.7) (2021-11-03)

- 🥚 patches update. fixes #2194 [`#2194`](https://github.com/open-wa/wa-automate-nodejs/issues/2194)
- 🥚 patches update. fixes #2257 [`#2257`](https://github.com/open-wa/wa-automate-nodejs/issues/2257)
- Release 4.23.7 [`3924189`](https://github.com/open-wa/wa-automate-nodejs/commit/392418909535737c447c29d4d20872a2b3db036b)
- 🐛  fix: #2264 [`aa736fd`](https://github.com/open-wa/wa-automate-nodejs/commit/aa736fd4faa3dd10c9bf1144250e7968b7e5f420)
- 🥚 patches update [`d50ccb7`](https://github.com/open-wa/wa-automate-nodejs/commit/d50ccb74c328fd5a22064d1be89622b3e0c62607)
- 🔊 logs: added more specific logs to initializer #2250 [`7c2cd91`](https://github.com/open-wa/wa-automate-nodejs/commit/7c2cd9199f3229e1cdb2d869495a33ac5490b2ef)
- updated types-only package [`0441ec4`](https://github.com/open-wa/wa-automate-nodejs/commit/0441ec4e70419012389755f684d05c860b7cfdc9)
